source/reference/minio-mc/mc-ilm-tier-add.rst

```diff
@@ -0,0 +1,343 @@
+
+- :policy-action:`admin:SetTier`
+- :policy-action:`admin:ListTier`
+
+For example, the following policy provides permission for configuring object transition lifecycle management rules on any bucket in the cluster: 
+
+.. literalinclude:: /extra/examples/LifecycleManagementAdmin.json
+   :language: json
+   :class: copyable
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following example creates a new remote tier called ``WARM-MINIO-TIER`` on the ``myminio`` deployment.
+      The command creates a tier for a remote MinIO deployment located at the hostname ``https://warm-minio.com``.
+      
+      .. code-block:: shell
+         :class: copyable
+
+          mc ilm tier add minio myminio WARM-MINIO-TIER                     \
+                                        --endpoint https://warm-minio.com   \                       
+                                        --access-key ACCESSKEY              \
+                                        --secret-key SECRETKEY              \
+                                        --bucket mybucket                   \
+                                        --prefix myprefix/ 
+
+      Lifecycle management rules on the ``myminio`` deployment can use the new tier to transition objects into the remote location's ``myprefix/`` prefix in the ``mybucket`` bucket.
+
+   .. tab-item:: SYNTAX
+   
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc ilm tier add TIER_TYPE                    \
+                         TARGET                       \
+                         TIER_NAME                    \
+                         --endpoint string            \
+                         [--region string]            \
+                         [--access-key value]         \
+                         [--secret-key value]         \
+                         [--use-aws-role]             \
+                         [--account-name value]       \
+                         [--account-key value]        \
+                         [--credentials-file value]   \
+                         --bucket value               \
+                         [--prefix value]             \
+                         [--storage-class value]
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+
+Parameters
+~~~~~~~~~~
+
+The command accepts the following arguments:
+
+.. mc-cmd:: TIER_TYPE
+   :required:
+
+   The Cloud Service Provider storage backend ("Tier") to which MinIO transitions objects. 
+   Specify *one* of the following supported values:
+
+   .. list-table::
+      :stub-columns: 1
+      :width: 100%
+      :widths: 30 70
+
+      * - ``minio``
+        - Use a remote MinIO deployment as the storage backend for the new Tier.
+
+          Requires specifying the following additional options:
```

Comment:
For all other instances:

"Requires specifying the following parameters"

Reply:
👍